include/zephyr/bluetooth/conn.h

```diff
@@ -1442,6 +1482,23 @@ int bt_conn_auth_info_cb_unregister(struct bt_conn_auth_info_cb *cb);
  */
 int bt_conn_auth_passkey_entry(struct bt_conn *conn, unsigned int passkey);
 
+/** @brief Send Passkey Keypress Notification during pairing
+ *
+ *  This function may be called only after passkey_entry callback from
+ *  bt_conn_auth_cb structure was called.
+ *
+ *  Requires @kconfig{CONFIG_BT_PASSKEY_KEYPRESS}.
+ *
+ *  @param conn Destination for the notification.
+ *  @param type What keypress event type to send. @see bt_conn_auth_keypress.
+ *
+ *  @retval 0 Success
+ *  @retval -EINVAL Improper use of the API.
```

Comment:
I don't think those comments are necessary, they're basically repeating the comments in errno.h